testing/tests/DevExpress.ui.widgets.editors/dropDownBox.tests.js

```diff
@@ -765,6 +767,19 @@ QUnit.module('popup options', moduleConfig, () => {
 });
 
 QUnit.module('keyboard navigation', moduleConfig, () => {
+    QUnit.test('alt+down should open dropDownBox', function(assert) {
```

Comment:
Unit test should be atomic: one test = 1 behavior check. U named this test as "alt+down should open dropDownBox", but u test 3 different points inside of it)

Let's leave only the third assert, but create a separate test if we need to test that dropDownBox is not opened after down key is pressed

Reply:
i dunno, checking that dropdown is closed before opening it and checking it is closed after some action seems natural to me, both assertions still stays in the context of the test as far as i'm concerned. But ok, single responsibility it is )